docs/pages/build-reference/infrastructure.mdx

```diff
@@ -146,14 +148,14 @@ Android workers run on Kubernetes in an isolated environment. Every build gets i
 
 ## iOS build server configurations
 
-iOS worker VMs run on Mac Mini 8.1 hosts in an isolated environment. Every build gets its own fresh macOS VM:
+iOS worker VMs run on Mac Mini 8.1 hosts in an isolated environment. Every build gets its own fresh macOS VM. We run a maximum of two builder VMs per one M1 host at the same time:
 
 - Hardware:
   - Intel: Intel(R) Core(TM) i7-8700B CPU (6 cores/12 threads), 64 GB RAM
-  - M1: 3.2GHz 8-Core M1, 16GB RAM [2020]
-- Build resource limits:
-  - Intel: 3 cores, 12 GB RAM
-  - M1: 3 cores, 8 GB RAM
+  - M1: 3.2GHz 8-Core M1 (4 performance and 4 efficiency cores), 16 GB RAM [2020]
+- Build resources:
+  - [`intel-medium`](eas-json/#resourceclass-2): 3 cores, 12 GB RAM
+  - [`m1-medium`](eas-json/#resourceclass-2): 2 cores, 8 GB RAM
```

Comment:
@wkozyra95 What do you think about it? I added information about the fact that we run max 2 VMs per host.

Reply:
I would expect it in this line where the medium class is defined, but if we don't have a large class year it's probably fine for now.